test/local/signer_tests.js

```diff
@@ -1,287 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-var test = require('../ptaptest')
-var path = require('path')
-var CC = require('compute-cluster')
-var bidcrypto = require('browserid-crypto')
-require('browserid-crypto/lib/algs/rs')
-var config = require('../../config').root()
-var signer = new CC({ module: path.join(__dirname, '../signer-stub.js')})
-signer.on('error', function () {}) // don't die
-
-var validKey = {
-  algorithm: 'RS',
-  n: '4759385967235610503571494339196749614544606692567785790953934768202714280652973091341316862993582789079872007974809511698859885077002492642203267408776123',
-  e: '65537'
-}
-
-test(
-  'returns an error on undefined publicKey',
-  function (t) {
-    signer.enqueue(
-      {
-        email: 'test@example.com',
-        duration: 100
-      },
-      function (err, result) {
-        t.ok(result.err, 'undefined publicKey')
-        t.end()
-      }
-    )
-  }
-)
-
-test(
-  'returns an error on empty publicKey',
-  function (t) {
-    signer.enqueue(
-      {
-        email: 'test@example.com',
-        publicKey: {},
-        duration: 100
-      },
-      function (err, result) {
-        t.equal(result.err.message, 'no such algorithm: undefined', 'no algorithm')
-        t.end()
-      }
-    )
-  }
-)
-
-test(
-  'returns and error on publicKey with missing parameters',
-  function (t) {
-    signer.enqueue(
-      {
-        email: 'test@example.com',
-        publicKey: {
-          algorithm: 'RS',
-          n: '1234'
-        },
-        duration: 100
-      },
-      function (err, result) {
-        t.ok(result.err.message.indexOf('bad key') === 0, 'bad key')
-        t.end()
-      }
-    )
-  }
-)
-
-test(
-  'returns and error on publicKey with bad parameters',
-  function (t) {
-    signer.enqueue(
-      {
-        email: 'test@example.com',
-        publicKey: {
-          algorithm: 'RS',
-          n: '7',
-          e: '65537'
-        },
-        duration: 100
-      },
-      function (err, result) {
-        t.ok(result.err.message.indexOf('bad key') === 0, 'bad key')
-        t.end()
-      }
-    )
-  }
-)
-
-test(
-  'returns an error with a bad duration',
-  function (t) {
-    signer.enqueue(
-      {
-        email: 'test@example.com',
-        publicKey: validKey,
-        duration: -1
-      },
-      function (err, result) {
-        t.equal(result.err.message, 'bad duration', 'bad duration')
-        t.end()
```

Comment:
I guess a lot of these tests are no longer required if you're not using computer-cluster, but ISTM at this least "bad duration" one should still show up somewhere.